server/database/query.go

```diff
@@ -0,0 +1,19 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func CheckObjectIsPlanet(db *sql.DB, id int64) (bool, error) {
+	rows, err := db.Query("select * from t_object join t_planet on t_planet.id = t_object.id where t_object.id = ?", id)
```

Comment:
this may be queryRow and you want to select something specific rather then everything because you read only the owner

Reply:
If we are doing this often we can fetch all the planets to the map and then check presence in the map locally

Reply:
Updated to use QueryRow with select count(*) instead of select *. For now, it's used only in this case. We can extract to the map later.

Reply:
unresolving - you are correct, for multiple calls this would be unefficient. Let's query all planets and compare IDs with it.

Reply:
updated the code, please take another look